mobile/src/notifications/notification-listener.ts

```diff
@@ -0,0 +1,68 @@
+/******************************************************************************
```

Comment:
Why the name of this file is "notification-listener.ts"? All files with epics in this project have names like "something.epics.ts". Consider renaming to "notifications.epics.ts"